core/builtins/message/__init__.py

```
@@ -140,14 +157,18 @@ def __str__(self):
 
 class MessageSession:
+    """
+    消息会话。
+    """
+
     def __init__(self,
                  target: MsgInfo,
                  session: Session):
         self.target = target
         self.session = session
         self.sent: List[MessageChain] = []
```

Ruff (0.8.2)

`MessageChain` may be undefined, or defined from star imports

(F405)